test/index.test.js

```diff
@@ -330,62 +330,105 @@ describe('Delete all eventmetadata', () => {
     expect(res.id).toBe(248723)
-    expect(res.status).toBe('VERIFIED')
+    expect(res.webhook_status).toBe('verified')
+    expect(res.enabled).toBe(true)
   })
-  it('Bad request error on create webhook registration', async () => {
-    const api = 'createWebhookRegistration'
+  it('Bad request error on create registration', async () => {
+    const api = 'createRegistration'
     exponentialBackoffMockReturnValue({}, { status: 400, statusText: 'Bad Request' })
-    await checkErrorResponse(api, new errorSDK.codes.ERROR_CREATE_REGISTRATION(), ['consumerOrgId', 'integrationId', mock.data.createWebhookRegistrationBadRequest])
+    await checkErrorResponse(api, new errorSDK.codes.ERROR_CREATE_REGISTRATION(), ['consumerId', 'projectId', 'workspaceId', mock.data.createRegistrationBadRequest])
   })
 })
 
-describe('Get all webhook registration', () => {
-  it('Success on get all webhook registration', async () => {
+describe('Update registration', () => {
+  it('Success on update registration', async () => {
     const sdkClient = await createSdkClient()
-    exponentialBackoffMockReturnValue(mock.data.getAllWebhookRegistrationsResponse, { status: 200, statusText: 'OK' })
-    const res = await sdkClient.getAllWebhookRegistrations('consumerOrgId', 'integrationId')
-    expect(res.length).toBe(2)
-    expect(res[0].id).toBe(1)
-    expect(res[1].status).toBe('VERIFIED')
+    exponentialBackoffMockReturnValue(mock.data.updateRegistrationResponse, { status: 200, statusText: 'OK' })
+    const res = await sdkClient.updateRegistration('consumerId', 'projectId', 'workspaceId', 'registrationId', mock.data.updateRegistration)
+    expect(res.id).toBe(248723)
+    expect(res.webhook_status).toBe('verified')
+    expect(res.delivery_type).toBe('webhook_batch')
+    expect(res.enabled).toBe(true)
+  })
+  it('Bad request error on update registration', async () => {
+    const api = 'updateRegistration'
+    exponentialBackoffMockReturnValue({}, { status: 400, statusText: 'Bad Request' })
+    await checkErrorResponse(api, new errorSDK.codes.ERROR_UPDATE_REGISTRATION(), ['consumerId', 'projectId', 'workspaceId', 'registrationId', mock.data.createRegistrationBadRequest])
   })
-  it('Not found error on get all webhook registration', async () => {
-    const api = 'getAllWebhookRegistrations'
+})
+
+describe('Get all registration', () => {
+  it('Success on get all registration', async () => {
+    const sdkClient = await createSdkClient()
+    exponentialBackoffMockReturnValue(mock.data.getAllRegistrationsResponse, { status: 200, statusText: 'OK' })
+    const res = await sdkClient.getAllRegistrationsForWorkspace('consumerId', 'projectId', 'workspaceId')
+    expect(res._embedded.registrations.length).toBe(3)
+    const regs = res._embedded.registrations
+    expect(res._links.self.href).toBe('https://api.adobe.io/events/consumerId/projectId/workspaceId/registrations')
```

Comment:
Can we create a constant for the baseURL (api.adobe.io/events) here?

Reply:
done